code/__HELPERS/mobs.dm

```diff
@@ -943,6 +943,77 @@ GLOBAL_DATUM_INIT(dview_mob, /mob/dview, new)
 		. = invoked_callback.Invoke()
 	usr = temp
 
+/proc/species_name_to_hud(species)
+	species = lowertext(species) // sanitize your text
+	if(species == "human")  // gotta *love* species code!!
+		return "species_human"
+
+	// theres so many repeats here to try and catch every variant for species players would enter
+	var/static/list/lookup = list(
+		"cat" 			= /datum/species/human/felinid,
```

Comment:
This entire proc seems like garbage to me. Only use the names the code ever actually uses in game.